python/tests/res/enkf/data/test_custom_kw.py

```diff
@@ -133,6 +138,42 @@ def test_custom_kw_set_values(self):
         self.assertEqual(ckw["STRING"], "string")
         self.assertEqual(ckw["FLOAT"], 3.1415)
         self.assertEqual(ckw["INT"], 1)
-
-
-
+    def test_custom_kw_complete_internalization(self):
+       root_dir = self.createTestPath('local/custom_kw')
+       config_file = 'custom_kw/minimal.ert'
+       with TestAreaContext('custom_kw_internalization', store_area=True) as work_area:
+           work_area.copy_directory(root_dir)
+            res_config = ResConfig(user_config_file=config_file)
```

Comment:
indentation seems messed up

Reply:
I know, but Citrix went down the minute after I pushed it. Will fix it later tonight 😉